test/hub/integration/Cases.t.sol

```diff
@@ -103,152 +103,152 @@ contract TestCases is BaseTest {
-    }
-
-    /// forge-config: default.isolate = true
-    function testRedeem() public returns (PoolId poolId, ShareClassId scId) {
-        (poolId, scId) = testDeposit();
-
-        cv.requestRedeem(poolId, scId, USDC_C2, INVESTOR, SHARE_AMOUNT);
-
-        uint128 revokedAssetAmount = PricingLib.convertWithPrice(
-            APPROVED_SHARE_AMOUNT, hubRegistry.decimals(poolId), hubRegistry.decimals(USDC_C2), NAV_PER_SHARE
-        );
-
-        vm.startPrank(FM);
-        hub.callRequestManager(
-            poolId,
-            CHAIN_CV,
-            abi.encodeCall(
-                IHubRequestManager.approveRedeems,
-                (
-                    poolId,
-                    scId,
-                    USDC_C2,
-                    hubRequestManager.nowRedeemEpoch(scId, USDC_C2),
-                    APPROVED_SHARE_AMOUNT,
-                    d18(1, 1)
-                )
-            )
-        );
-        hub.callRequestManager(
-            poolId,
-            CHAIN_CV,
-            abi.encodeCall(
-                IHubRequestManager.revokeShares,
-                (poolId, scId, USDC_C2, hubRequestManager.nowRevokeEpoch(scId, USDC_C2), NAV_PER_SHARE, HOOK_GAS)
-            )
-        );
-
-        // Queue cancellation request which is fulfilled when claiming
-        cv.cancelRedeemRequest(poolId, scId, USDC_C2, INVESTOR);
-
-        vm.startPrank(ANY);
-        vm.deal(ANY, GAS);
-        hub.notifyRedeem{value: GAS}(
-            poolId, scId, USDC_C2, INVESTOR, hubRequestManager.maxRedeemClaims(scId, INVESTOR, USDC_C2)
-        );
-
-        MessageLib.RequestCallback memory m0 = MessageLib.deserializeRequestCallback(cv.popMessage());
-        assertEq(m0.poolId, poolId.raw());
-        assertEq(m0.scId, scId.raw());
-        assertEq(m0.assetId, USDC_C2.raw());
-
-        RequestCallbackMessageLib.RevokedShares memory cb0 =
-            RequestCallbackMessageLib.deserializeRevokedShares(m0.payload);
-        assertEq(cb0.assetAmount, revokedAssetAmount);
-        assertEq(cb0.shareAmount, APPROVED_SHARE_AMOUNT);
-        assertEq(cb0.pricePoolPerShare, NAV_PER_SHARE.raw());
-
-        MessageLib.RequestCallback memory m1 = MessageLib.deserializeRequestCallback(cv.popMessage());
-        assertEq(m1.poolId, poolId.raw());
-        assertEq(m1.scId, scId.raw());
-        assertEq(m1.assetId, USDC_C2.raw());
-
-        RequestCallbackMessageLib.FulfilledRedeemRequest memory cb1 =
-            RequestCallbackMessageLib.deserializeFulfilledRedeemRequest(m1.payload);
-        assertEq(cb1.investor, INVESTOR);
-        assertEq(cb1.fulfilledAssetAmount, revokedAssetAmount);
-        assertEq(cb1.fulfilledShareAmount, APPROVED_SHARE_AMOUNT);
-        assertEq(cb1.cancelledShareAmount, SHARE_AMOUNT - APPROVED_SHARE_AMOUNT);
-    }
+    // /// forge-config: default.isolate = true
+    // function testDeposit() public returns (PoolId poolId, ShareClassId scId) {
```

Comment:
I think we can remove those. EndToEnd should cover it. It's true that this ensures each parameter in the messages are correct, but the investment process can not work if it's not correct... Not sure what others think about this